packages/esm-outpatient-app/src/config-schema.ts

```diff
@@ -81,6 +81,10 @@ export const configSchema = {
       _type: Type.ConceptUuid,
       _default: '1343AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
     },
+    visitAttendeeConceptUuid: {
```

Comment:
A description here would be helpful to in knowing what it does

Reply:
@CynthiaKamau  Please make the implementation generic as we have discussed.

Reply:
Done